pkg/scheduler/metrics/metrics.go

```diff
@@ -142,6 +142,13 @@ var (
 			Buckets:        metrics.LinearBuckets(5, 5, 10),
 			StabilityLevel: metrics.ALPHA,
 		})
+	PreemptionVictimsCount = metrics.NewCounter(
+		&metrics.CounterOpts{
```

Comment:
We used to have a `Gauge` type metric, but it's changed to histogram in #83603.

So @liu-cong I don't think we want to change it back, right?

Reply:
No we don't want Gauge. I think that change is to record the victims each scheduling cycle

Reply:
Yes, it is. Thank you for clarifying.